src/components/UI/AttachmentLink/AttachmentLink.tsx

```diff
@@ -100,13 +99,15 @@ export const AttachmentLink = ({ filename, data, type, path }: AttachmentLinkPro
   filesize = prettyBytes(decodedData.length);
 
   return (
-    <Attachment>
+    <div className="transition duration-300 ease-out flex-1 border border-solid border-grey-200 py-2 px-4 hover:no-underline hover:bg-blue-300">
       <div className="flex flex-row">
         <div className="w-auto mr-4">{getExtension(type)}</div>
         <div className="w-5/6">
           <p className="mb-2 break-all">
-            <span className="filename">{filename}</span>
-            {hasBase64 && <span className="font-normal inline-block text-grey text-sm">&nbsp;({filesize})</span>}
+            <span className="transition duration-300 ease-out leading-5 text-grey-700 break-words font-semibold">
+              {filename}
+            </span>
+            {hasBase64 && <span className="font-normal text-grey text-sm">&nbsp;({filesize})</span>}
```

Comment:
its probably not you but i would rather use css to adjust the spacing rather than use escaped space (&nbsp) to give a spacing